tests/src/test/java/alluxio/hadoop/FileSystemRenameIntegrationTest.java

```diff
@@ -223,7 +223,7 @@ public void basicRenameTest7() throws Exception {
     sTFS.mkdirs(dirA);
     FSDataOutputStream o = sTFS.create(fileA);
     o.writeBytes("Test Bytes");
-    o.sync();
+    o.hflush();
```

Comment:
it's probably best to suppress the deprecation warning and say why we use the deprecated method